mizu/kernel/Makefile

```diff
@@ -7,7 +7,9 @@ KERNEL_ASM  := $(DEBUG_DIR)/$(KERNEL).asm
 
 CARGO_ARGS := --target $(TARGET) \
 	--no-default-features \
-	--features $(BOARD)
+	--features $(BOARD) \
+	-Zbuild-std=core,compiler_builtins,alloc,panic_abort \
```

Comment:
Moved `build-std` from config.toml to Makefile to limit it only in production builds, with unit tests getting rid of it.